arrows/ocv/resection_camera.cxx

```diff
@@ -0,0 +1,218 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+/// \file
+/// \brief OCV resection_camera algorithm implementation
+
+#include "camera_intrinsics.h"
+#include "resection_camera.h"
+
+#include <opencv2/calib3d/calib3d.hpp>
+#include <opencv2/core/eigen.hpp>
+
+using namespace std;
+using namespace cv;
+
+namespace kwiver {
+
+namespace arrows {
+
+namespace ocv {
+
+// ----------------------------------------------------------------------------
+
+struct resection_camera::priv
+{
+  double reproj_accuracy = 4.;
+  int max_iterations = 32;
+  vital::logger_handle_t m_logger;
+
+  priv() : m_logger{ vital::get_logger( "arrows.ocv.resection_camera" ) }
+  {
+  }
+};
+
+// ----------------------------------------------------------------------------
+
+resection_camera
+::resection_camera()
+  : d_{ new priv }
+{
+}
+
+resection_camera::~resection_camera()
+{
+}
+
+// ----------------------------------------------------------------------------
+vital::config_block_sptr
+resection_camera
+::get_configuration() const
+{
+  // get base config from base class
+  vital::config_block_sptr config =
+    vital::algo::resection_camera::get_configuration();
+  config->set_value( "reproj_accuracy", d_->reproj_accuracy,
+                     "desired re-projection positive accuracy" );
+  config->set_value( "max_iterations", d_->max_iterations,
+                     "maximum number of iterations to run PnP [1, INT_MAX]" );
+  return config;
+}
+
+/// Set this algorithm's properties via a config block.
+void
+resection_camera
+::set_configuration( vital::config_block_sptr config )
+{
+  d_->reproj_accuracy = config->get_value< double >( "reproj_accuracy",
+                                                     d_->reproj_accuracy );
+  d_->max_iterations = config->get_value< int >( "max_iterations",
+                                                 d_->max_iterations );
+}
+
+/// Check that the algorithm's configuration vital::config_block is valid.
+bool
+resection_camera
+::check_configuration( vital::config_block_sptr config ) const
+{
+  bool good_conf = true;
+  double reproj_accuracy = config->get_value< double >( "reproj_accuracy",
+                                                        d_->reproj_accuracy );
```

Comment:
```suggestion
  auto const reproj_accuracy =
    config->get_value< double >( "reproj_accuracy", d_->reproj_accuracy );
```

...or `double const`, the suggestion is mostly adding `const` (and secondarily, wrapping differently since the `const` otherwise makes this too long).

Likewise for `max_iterations`, below.